rcljava/src/main/java/org/ros2/rcljava/timer/WallTimerImpl.java

```diff
@@ -17,113 +17,19 @@
 
 import java.lang.ref.WeakReference;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import org.ros2.rcljava.common.JNIUtils;
 import org.ros2.rcljava.concurrent.Callback;
 import org.ros2.rcljava.node.Node;
+import org.ros2.rcljava.timer.TimerImpl;
+import org.ros2.rcljava.timer.WallTimer;
 
-public class WallTimerImpl implements WallTimer {
-  private static final Logger logger = LoggerFactory.getLogger(WallTimerImpl.class);
-
-  static {
-    try {
-      JNIUtils.loadImplementation(WallTimerImpl.class);
-    } catch (UnsatisfiedLinkError ule) {
-      logger.error("Native code library failed to load.\n" + ule);
-      System.exit(1);
-    }
-  }
-
-  private long timerPeriodNS;
-
-  private final WeakReference<Node> nodeReference;
-
-  private long handle;
-
-  private final Callback callback;
-
-  private static native boolean nativeIsReady(long handle);
-
-  private static native boolean nativeIsCanceled(long handle);
-
-  private static native long nativeTimeSinceLastCall(long handle);
-
-  private static native long nativeTimeUntilNextCall(long handle);
-
-  private static native long nativeReset(long handle);
-
-  private static native long nativeCancel(long handle);
-
-  private static native long nativeGetTimerPeriodNS(long handle);
-
-  private static native void nativeSetTimerPeriodNS(long handle, long period);
-
-  private static native long nativeCallTimer(long handle);
-
+/**
+ * @deprecated
+ * Use @{link TimerImpl} instead.
+ */
+@Deprecated
+public class WallTimerImpl extends TimerImpl implements WallTimer {
```

Comment:
I think `*Impl` classes are never used directly, so this probably can be deleted instead of deprecated.

Reply:
I figured it is a public class, so technically someone could be using it.